zest/releaser/utils.py

```diff
@@ -695,8 +695,8 @@ def _execute_command(command):
         "stderr": subprocess.PIPE,
         "env": env,
     }
-    with subprocess.Popen(command, **process_kwargs) as process:
-        return _subprocess_open(process, command, show_stderr)
+    process = subprocess.run(command, **process_kwargs)
+    return _subprocess_open(process, show_stderr)
```

Comment:
I'm wondering a bit about still using the `_subprocess_open()` name. Perhaps it is just a naming thing.
And probably you've fixed it in another commit (I'm going through everything commit-by-commit as you suggested :-) )